hasktorch/src/Torch/Data/StreamedPipeline.hs

```diff
@@ -75,37 +75,24 @@ pmap' n f  prod = ContT $ \cont ->
       (\output -> runEffect $ enumerate prod >-> f >-> toOutput output)
       (\input -> cont . Select $ fromInput input)
 
+
 makeListT ::
   forall batch m dataset seed b r.
   (Datastream m seed dataset batch, MonadBaseControl IO m, MonadBase IO m) =>
```

Comment:
```suggestion
  (Datastream m seed dataset a, MonadBaseControl IO m, MonadBase IO m) =>
```
again, I'm not sure it's necessary for these to be batches. What do you think?

Reply:
I agree, I changed the parameter to `sample` to be in line with the map style datasets.